EIPS/eip-1.md

```diff
@@ -55,7 +55,7 @@ Each status change is requested by the EIP author and reviewed by the EIP editor
 * **Draft** -- Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the EIP to be mature and ready to proceed to the next status. An EIP in draft status must be implemented to be considered for promotion to the next status (ignore this requirement for core EIPs).
   * :arrow_right: Last Call -- If agreeable, the EIP editor will assign Last Call status and set a review end date (`review-period-end`), normally 14 days later.
   * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that EIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
-* **Last Call** -- This EIP will listed prominently on the http://eips.ethereum.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
+* **Last Call** -- This EIP will listed prominently on the https://eips.ethereum.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
```

Comment:
This was here originally but should be changed in a separate PR to
```suggestion
* **Last Call** -- This EIP will be listed prominently on the https://eips.ethereum.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
```
I have done too much editing and other work for Ethereum without getting remunerated for it so can't do that further.

Reply:
I will make this change in a separate PR. (To keep this one pure-play.)

No date promised. Will credit you. I have other PRs outstanding against EIP1 already, hoping to resolve them first.